test/gui/shared/steps/steps.py

```diff
@@ -724,3 +735,19 @@ def step(context, resource, group):
 
     sharingDialog = SharingDialog()
     sharingDialog.selectCollaborator(group, True)
+
+
+@When('the user overwrites the file "|any|" with content "|any|"')
+def step(context, resource, content):
+    print("before overwrite")
+    waitForFileToBeSynced(context, resource)
+    waitForFolderToBeSynced(context, '/')
+
+    snooze(3)
```

Comment:
can you write a comment above the `snooze(3)` that explains why this static wait is necessary?

Reply:
That's a good idea. I'll add a comment and push it.

Reply:
the problem is described in https://github.com/owncloud/client/issues/8832

It is an artificial delay set in the client. See also https://github.com/owncloud/client/blob/9117275c67ca311152b4f16749c6fd32c3d40fe9/src/libsync/syncengine.cpp#L61-L74

Reply:
Please mention #8832 in the comment, that gives more background